src/mbpfan.c

```diff
@@ -52,6 +53,7 @@
 
 #define CORETEMP_PATH "/sys/devices/platform/coretemp.0"
 #define APPLESMC_PATH "/sys/devices/platform/applesmc.768"
+#define ALT_APPLESMC_PATH "/sys/bus/acpi/drivers/applesmc"
```

Comment:
Is this the new, preferred path, or just some weird quirk of Debian/Ubuntu?

Reply:
On the MacBookPro11,3, `/sys/devices/platform/applesmc.768` is still being used, and `/sys/bus/acpi/drivers/applesmc` is empty, but since that is a mid-2014 machine, things may be different for newer macs.

(I will say that the stock `mbpfan` package available in ubuntu 22.04 was not working to set the fan speed on my MacBookPro11,3 though, whereas building with this patch does set the fan speeds according to my settings in the `mbpfan.conf` file.)

Reply:
(so maybe one of the other changes... or doing a rebuild for 22.04 made the package work.)

Reply:
I tested my MacBook Air 2014 with 5.16.8-200.fc35.x86_64 and `/sys/devices/platform/applesmc.768` has the files that `mbpfan` needs.  Does kernel 5.17 change this path?  Can you point me to the Linux commit that changes this?  The original PR talks about kernel 5.13 -- what am I missing here?